Source/Core/VideoBackends/OGL/TextureCache.cpp

```diff
@@ -293,7 +294,17 @@ TextureCache::TextureCache()
 
 	if (g_ActiveConfig.backend_info.bSupportsPaletteConversion)
 	{
-		s_palette_stream_buffer = StreamBuffer::Create(GL_TEXTURE_BUFFER, 1024*1024);
+		s32 buffer_size = 1024 * 1024;
+		s32 max_buffer_size = 0;
+
+		// The minimum MAX_TEXTURE_BUFFER_SIZE that the spec mandates
+		// is 65KB, we are asking for a 1MB buffer here.
```

Comment:
Actually the spec says (a) 64K, not 65K, and (b) never talks about bytes at all, but rather texels.